acceptance/Rakefile

```diff
@@ -16,6 +16,9 @@ CLEAN.include('*.tar', REPO_CONFIGS_DIR, 'merged_options.rb')
 # then assume we can depend on them here
 EPEL_MIRROR = ENV['BEAKER_EPEL_MIRROR']
 
+# Default test target if none specified
+DEFAULT_TEST_TARGET = 'centos7-64a-windows2012r2-64a'
```

Comment:
We were talking about this in the context of puppetserver acceptance, where they run an agent on the master. What do you think about defaulting to that configuration (as opposed to one master and two different agents)? That way we have a workflow for making sure we don't break them with each new acceptance test.

It does make the logic a little more complicated, since you'd have to add the `agent` role to the `MASTER_TEST_TARGET`... thoughts?